app/controllers/courses_controller.rb

```diff
@@ -0,0 +1,32 @@
+class CoursesController < ApplicationController
+  before_filter :load_course, :except => [:index]
+
+  def index
+    @courses = Course.paginate(page: params[:page], per_page: DEFAULT_PER_PAGE).decorate
+  end
+
+  def show
+  end
+
+  def edit
+  end
+
+  def update
+    if @course.update_attributes(params[:course])
+      redirect_to(@course)
+    else
+      render :action => "edit"
```

Comment:
hash rocket here, but also I prefer symbol over a string here

```suggestion
      render action: :edit```